nin/backend/fontgen.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs-promise');
+const glob = require('glob');
+const path = require('path');
+
+
+async function fontGen(pathPrefix) {
+
+  let resolver;
+  let promise = new Promise(resolve => {
```

Comment:
I think it is equivalent to wrapping the glob inside the promise callback. I split it like this to reduce indentation.